qcodes/instrument_drivers/rohde_schwarz/private/HMC804x.py

```diff
@@ -1,37 +1,46 @@
-from qcodes import VisaInstrument, validators as vals
+from typing import Union
+
+from qcodes import VisaInstrument, validators as vals, Instrument
 from qcodes import InstrumentChannel, ChannelList
 
 
 class RohdeSchwarzHMC804xChannel(InstrumentChannel):
-    def __init__(self, parent, name, channel):
+    def __init__(self, parent: Instrument, name: str,
+                 channel: Union[str, int]) -> None:
         super().__init__(parent, name)
 
         select_cmd = ":INSTrument:NSELect {};".format(channel)
 
         self.add_parameter("set_voltage",
                            label='Target voltage output',
-                           set_cmd="{} :SOURce:VOLTage:LEVel:IMMediate:AMPLitude {}".format(
-                               select_cmd, '{}'),
-                           get_cmd="{} :SOURce:VOLTage:LEVel:IMMediate:AMPLitude?".format(
-                               select_cmd),
+                           set_cmd="{} "
+                                   ":SOURce:VOLTage:LEVel:IMMediate:AMPLitude "
+                                   "{}".format(select_cmd, '{}'),
+                           get_cmd="{} "
+                                   ":SOURce:VOLTage:LEVel:IMMediate:AMPLitude?".
+                                   format(select_cmd),
```

Comment:
the version before diff was more readable :) i think a souluition here is to put `:SOURce:VOLTage:LEVel:IMMediate:AMPLitude` into a variable and reuse it in the cmd string?